_posts/2018-03-11-user-testing-warehouse.md

```diff
@@ -0,0 +1,54 @@
+
+Over three weeks, I ran tests with six different users, with each test taking between 25 minutes and one hour; the duration depending on how much the test subject had to say.
+
+As expected, the tests highlighted a number of problems in the UI, which in turn prompted me to open or tag [20 issues](https://github.com/pypa/warehouse/issues?q=is%3Aissue+label%3A%22user+testing%22) in our issue tracker. The vast majority of these tickets are simple changes that have been tagged as [good first issues](https://github.com/pypa/warehouse/issues?q=is%3Aissue+label%3A%22good+first+issue%22) - which should in turn help with our efforts to outreach to new contributors.
+
+Several of the test participants also raised new ideas, for which I will open new tickets soon.
+
+A big thanks to Ernest W. Durbin III, Nicolas Dubois, Łukasz Langa, Russell Keith-Magee, Alex Chan and Eliot Berriot for participating in these tests.
+
+## Next Steps
+
+From this intial program it is clear that user testing can bring value to open source projects. The challenge is to continue to conduct such tests once our MOSS funding is depleted.
+
+Fortunately, the Python community is engaged with the Warehouse project, and, so far, we've had no problems recruiting test participants. Usability experts [recommend a minumum of 5 users per test](https://www.nngroup.com/articles/how-many-test-users/), so as long as this goodwill continues, I don't anticipate running out of test subjects.
+
+On the other hand, setting up and running user tests takes time; and it would be nice to have some help with this to lighten the workload (and increase our testing bus factor). For this reason, today I've put out a call to [train up to three people on how to conduct and run user tests](TODO). My hope is that this opportunity will be taken by students (or UX enthusiasts) as a way to build their experience by working on a live, widely used application.
+
+Should this program be successful, I'd like to start group testing the [project detail page](https://pypi.org/project/Flask/) in the coming weeks. This page is the most important on the site, and coincidentally, the page we've also received the most feedback about.
+
+## Get Involved
+
+So, how can you help?
+
+Firstly, if you are a developer, please consider contributing to [Warehouse](https://github.com/pypa/warehouse). Right now, we have lots of tickets marked as [good first issues](https://github.com/pypa/warehouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) in our [issue tracker](https://github.com/pypa/warehouse/issues). Because of our MOSS funding, we are also able to be more responsive to your questions and provide you with more support to get started.
```

Comment:
Perhaps link to https://wiki.python.org/psf/PackagingSprints and mention we'll be at some conferences, as well as linking to the Getting Started guide https://warehouse.readthedocs.io/development/getting-started/ ?